shared/xccdf/services/obsolete.xml

```diff
@@ -172,6 +196,20 @@ Enterprise Linux.</rationale>
 <ref cis="2.1.2" cui="3.1.13" />
 </Rule>
 
+<!-- not using yum macro but apt. shared oval file should be overloaded by local debian/ubuntu csv -->
+<Rule id="package_telnet_removed" severity="low" prodtype="debian8,ubuntu1404">
```

Comment:
If this rule already exists, it should be collapsed into an existing rule and the appropriate elements updated with `prodtype`